File: compare.py
from cs50 import get_int

x = get_int ("what's x? ")
y = get_int ("what's y? ")

if (x < y):
    print("x is less than y")
elif (x > y):
    print("x is gretar than y")
else:
    print("x is equal to y")

File: compare.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
 int x= get_int("what's x ");
 int y= get_int("what's y ");

 if (x < y)
 {
    printf("x is lessthan y\n");
 }
 else if (x>y)
 {
    printf("x is greaterthan y\n");
}
else
   {
   printf ("x is eqaul to y\n");
}
}
